ui/src/assets/i18n/fr.json

```diff
@@ -88,6 +88,7 @@
   "application_repoman_cant_link": "ici",
   "application_repo_title": "Lier l'application à un repository : ",
   "application_repo_strat_title" : "Configuration",
+  "application_deployment_title" : "Lier l'application à une plateforme de déploiement: ",
```

Comment:
key deployment_platform_name is missing (used in application -> advanced)

Reply:
done